frontend/e2e/specs/side-path/charting-unhappy-path.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { importModifiedCorrectConfig as modifiedCorrectProjectFromFile } from '../../fixtures/import-file/unhappy-path-file';
+import { chartUnHappyPathConfig } from '../../fixtures/create-new/config-step';
+import { test } from '../../fixtures/test-with-extend-fixtures';
+import { clearTempDir } from '../../utils/clear-temp-dir';
+import { format } from '../../utils/date-time';
+
+test.beforeAll(async () => {
+  await clearTempDir();
+});
+
+test('Charting unhappy path on config and metri page', async ({ homePage, configStep, metricsStep }) => {
+  const rightDateRange = {
+    startDate: format(modifiedCorrectProjectFromFile.dateRange.startDate),
+    endDate: format(modifiedCorrectProjectFromFile.dateRange.endDate),
+  };
+  const errorDateRange = {
+    startDate: format(chartUnHappyPathConfig.errorDateRange[0].startDate),
+    endDate: format(chartUnHappyPathConfig.errorDateRange[0].endDate),
+  };
+
+  const noCardDateRange = {
+    startDate: format(chartUnHappyPathConfig.noCardDateRange[0].startDate),
+    endDate: format(chartUnHappyPathConfig.noCardDateRange[0].endDate),
+  };
+  await homePage.goto();
+
+  await homePage.importProjectFromFile('../fixtures/input-files/charting-unhappy-path-config-file.json');
```

Comment:
I can't find any input file or template related to `charting-unhappy-path-config-file.json`, did I missed something?

Reply:
this json is not in change， and I have been put it